R/class.R

```diff
@@ -156,30 +156,20 @@ bake_check_class_core <- function(x,
   classes <- class(x)
   missing <- setdiff(class_nm, classes)
   if (length(missing) > 0) {
-    rlang::abort(
-      paste0(
-        var_nm,
-        " should have the class(es) ",
-        paste(class_nm, collapse = ", "),
-        " but has the class(es) ",
-        paste(classes, collapse = ", "),
-        "."
-      )
+    cli::cli_abort(
+      "{.var {var_nm}} should have the class{?es} {.and {.val {class_nm}}} but
```

Comment:
```suggestion
      "{.var {var_nm}} should have the class{?es} {.cls {class_nm}}} but
```

You may have intentionally not chosen the `.cls` markup. Thoughts?

Reply:
i forgot about this one class!